Tests/StoreKitUnitTests/StoreKit2/StoreKit2TransactionFetcherTests.swift

```diff
@@ -148,6 +148,22 @@ class StoreKit2TransactionFetcherTests: StoreKitConfigTestCase {
         expect(result) == transaction
     }
 
+    // MARK: - receipt
+
+    func testGeneratesReceipt() async throws {
+        try AvailabilityChecks.skipIfXcode14orEarlier()
```

Comment:
Even thought [subscriptionStatus](https://developer.apple.com/documentation/storekit/transaction/4137250-subscriptionstatus) (used in `receipt`) is back-deployed to iOS 15, it is not available in Xcode 14 :(

Reply:
Let's add a test to verify the behavior on Xcode 14 then.